### DISCUSSION TITLE
Changes in Home Assistant causing errors

### DISCUSSION THREAD

**Discussion by pdobrien3** [blakeblackshear/frigate#14055] [Report a Bug]
### Checklist

- [X] I have updated to the latest available Frigate version.
- [X] I have cleared the cache of my browser.
- [X] I have tried a different browser to see if it is related to my browser.
- [X] I have tried reproducing the issue in [incognito mode](https://www.computerworld.com/article/1719851/how-to-go-incognito-in-chrome-firefox-safari-and-edge.html) to rule out problems with any third party extensions or plugins I have installed.

### Describe the problem you are having

Logger: homeassistant.components.media_player.const
Source: helpers/deprecation.py:197
integration: Media player ([documentation](https://rc.home-assistant.io/integrations/media_player), [issues](https://github.com/home-assistant/core/issues?q=is%3Aissue+is%3Aopen+label%3A%22integration%3A+media_player%22))
First occurred: 4:36:32 PM (6 occurrences)
Last logged: 4:36:32 PM

MEDIA_CLASS_IMAGE was used from frigate, this is a deprecated constant which will be removed in HA Core 2025.10. Use MediaClass.IMAGE instead, please report it to the author of the 'frigate' custom integration
MEDIA_CLASS_MOVIE was used from frigate, this is a deprecated constant which will be removed in HA Core 2025.10. Use MediaClass.MOVIE instead, please report it to the author of the 'frigate' custom integration
MEDIA_CLASS_VIDEO was used from frigate, this is a deprecated constant which will be removed in HA Core 2025.10. Use MediaClass.VIDEO instead, please report it to the author of the 'frigate' custom integration
MEDIA_TYPE_IMAGE was used from frigate, this is a deprecated constant which will be removed in HA Core 2025.10. Use MediaType.IMAGE instead, please report it to the author of the 'frigate' custom integration
MEDIA_TYPE_VIDEO was used from frigate, this is a deprecated constant which will be removed in HA Core 2025.10. Use MediaType.VIDEO instead, please report it to the author of the 'frigate' custom integration

### Steps to reproduce

1. Home Assistant 2024.10.0b4
2.
3.
...

### Version

0.14.1-f4f3cfa

### Frigate config file

```yaml
N/A
```

### docker-compose file or Docker CLI command

```yaml
N/A
```

### Relevant Frigate log output

```shell
N/A
```

### Relevant go2rtc log output

```shell
N/A
```

### Operating system

Debian

### Install method

Docker Compose

### Network connection

Wired

### Camera make and model

Hikvision

### Screenshots of the Frigate UI's System metrics pages

N/A

### Any other information that may be helpful

N/A

**Comment by hawkeye217** [comment]
This should be reported on the [Frigate HA integration](https://github.com/blakeblackshear/frigate-hass-integration)'s repo. This repo is for Frigate itself.

Edit: The warning has already been fixed: https://github.com/blakeblackshear/frigate-hass-integration/pull/726
